.eslintignore

```diff
@@ -0,0 +1,10 @@
+# don't lint node_modules
+node_modules
+# don't lint build output
+dist
+# don't lint coverage output
+coverage
+# don't lint configuration
+karma.conf.js
+# don't lint spec.ts
+*.spec.ts
```

Comment:
you generally do want to lint test files.

Reply:
My reasoning for excluding *.spec.ts was because it seems like the linter doesn't understand its a test file, or, at least the wrong rules are being applied to them. I may be able to use an override to only apply or exclude a certain subset of the rules to spec.ts files. I'll look into it.

![Capture](https://user-images.githubusercontent.com/47096143/127056227-06fbdbf8-ce12-408a-b187-405dec5d28b9.PNG)

Reply:
ah. that file is relying on being run in a context where jasmine provides some global variables, which eslint isn't automatically aware of. try this https://stackoverflow.com/questions/36952252/eslint-angular-and-jasmine-is-not-defined-no-undef

Reply:
those answers do look old, though, so check the actual up-to-date docs with that as a guide

Reply:
Thanks for the info, that seemed to fix it.

Official docs say adding jasmine to the eslint environments adds all of the jasmine testing global variables for version 1.3 and 2.0.

Looking at the package.json, we're using jasmine 3.7 but I tested it, and it does not complain anymore about the spec.ts files so I guess the global variables haven't changed.